addons/web/static/src/views/kanban/kanban_arch_parser.js

```diff
@@ -104,13 +104,7 @@ export class KanbanArchParser extends XMLParser {
         let progressAttributes = false;
         const progressBar = xmlDoc.querySelector("progressbar");
         if (progressBar) {
-            const attrs = extractAttributes(progressBar, ["field", "colors", "sum_field", "help"]);
-            progressAttributes = {
-                fieldName: attrs.field,
-                colors: JSON.parse(attrs.colors),
-                sumField: fields[attrs.sum_field] || false,
-                help: attrs.help,
-            };
+            progressAttributes = this.parseProgressBar(progressBar, fields);
```

Comment:
Shouldn't these changes in "web" be placed in a separate commit?

Reply:
Sure